qa/L0_config_search/config_generator.py

```diff
@@ -212,4 +212,4 @@ def get_all_configurations(profile_models):
         with open(f'./config-{i}-models-remote.txt', 'w') as file:
             file.write(str(total_models_remote))
         with open(f'./config-{i}.yml', 'w') as file:
-            yaml.dump(configuration, file)
+            yaml.dump(configuration, file)
```

Comment:
New line.

Reply:
Please keep the new line at the end of all the files.

Reply:
Adjusted